src/main/scala/io/shiftleft/js2cpg/cpg/passes/astcreation/AstNodeBuilder.scala

```diff
@@ -393,7 +398,7 @@ class AstNodeBuilder[NodeBuilderType](private val diffGraph: DiffGraph.Builder,
     val code       = sanitizeCode(node)
     val block = NewBlock()
       .typeFullName(Defines.ANY.label)
-      .code(code)
+      .code(shortenCode(code))
```

Comment:
Can we not do this for the top-level block of `:program` methods please? That code is incredibly useful to me...

Reply:
Also after 1000 characters?

Reply:
I have added a switch for top-level program blocks.